src/main/java/org/broadinstitute/hellbender/utils/smithwaterman/SmithWatermanJavaAligner.java

```diff
@@ -24,6 +24,8 @@
 public final class SmithWatermanJavaAligner implements SmithWatermanAligner {
     private static final SmithWatermanJavaAligner ALIGNER = new SmithWatermanJavaAligner();
     private long totalComputeTime = 0;
+    private long totalHeuristicTime = 0;
```

Comment:
Similar to the above comment, I would not track this in the code.  We can easily measure the time saved by the heuristic with a profiler.